portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,22 +81,33 @@ private String convertToJsonUsingGson(ArrayList<String> comments){
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment");
     long timeStamp = System.currentTimeMillis();
-
-    if(text.length() != 0) {
-        comments.add(text);
-    }
+    
 
     Entity taskEntity = new Entity("Task");
     taskEntity.setProperty("comment", text);
     taskEntity.setProperty("timestamp", timeStamp);
+    
 
     datastore.put(taskEntity);
     
-    response.sendRedirect("/gallery.html");
+    response.sendRedirect("/index.html");
 
     
   }
-    
+
+  private int getNumberOfComments(HttpServletRequest request){
+    String parameterValue = request.getParameter("parameterValue");
+    System.out.println(parameterValue);
```

Comment:
It's good to remove this print statement if you don't intend to leave it in for future debugging.